Formula/mercury.rb

```diff
@@ -47,9 +42,18 @@ def install
       main(IOState_in, IOState_out) :-
           io.write_string("#{test_string}", IOState_in, IOState_out).
     EOS
+
     system "#{bin}/mmc", "--make", "hello"
     assert_predicate testpath/"hello", :exist?
 
     assert_equal test_string, shell_output("#{testpath}/hello")
+
+    rm "#{testpath}/hello"
```

Comment:
Can you just name them `hello_c` and `hello_java`?

Reply:
Good point. I can do `hello_c`, but `mmc` doesn't support arbitrary output names when compiling for Java.